wrappers/core18.go

```diff
@@ -98,19 +98,24 @@ WantedBy=snapd.service
 	if err != nil {
 		return err
 	}
-	if err := sysd.DaemonReload(); err != nil {
-		return err
+
+	if !opts.Preseeding {
+		if err := sysd.DaemonReload(); err != nil {
```

Comment:
Could the emulation mode DaemonReload be a nop then?

Reply:
Yes, I think it's a great idea.